workspace/python/ds-pack/2.1.0-base-python-3.10.13/vscode-tunnel/scripts/start.sh

```diff
@@ -0,0 +1 @@
+node /vscode-tunnel/scripts/init-or-start.js
```

Comment:
When we run the start script a second time, we see the logs from current as well as previous executions which makes it difficult to decipher what currently happened 🤷🏽 

<img width="1109" alt="Screenshot 2023-09-27 at 10 59 43 AM" src="https://github.com/peak-ai/platform-global-images/assets/12875685/25b50e58-1016-4559-9efc-c8e7a8a8b2f6">

Reply:
Hey tbh not sure if we can do much about it. I mean we can maybe trim the logs being displayed based on wether the tunnel is running or not but guess we'l have to add in some conditions to handle the same. Will see if something can be done real quick and easy.

Reply:
Have added in a small commit to limit the logs. Hope this helps to some extent.

Reply:
I highly doubt that this isn't doing much 🤔